src/privatesend.cpp

```diff
@@ -305,8 +306,9 @@ bool CPrivateSend::IsCollateralValid(const CTransaction& txCollateral)
 bool CPrivateSend::IsCollateralAmount(CAmount nInputAmount)
 {
     // collateral input should be smth between 1x and 2x OR exactly Nx
-    return (nInputAmount > GetCollateralAmount() && nInputAmount < GetCollateralAmount()*2) ||
-           (nInputAmount >= GetCollateralAmount() && nInputAmount <= GetMaxCollateralAmount() && nInputAmount % GetCollateralAmount() == 0);
+    int nMin = mnpayments.GetMinMasternodePaymentsProto() > 70208 ? 1 : 2;
```

Comment:
This won't disable the change in requirement for exact Nx collaterals. In the old version it was not allowed to have something between 1x and 2x as I understand it.

Reply:
Maybe it's better to do a simple `if (mnpayments.GetMinMasternodePaymentsProto() > 70208) {} else {}` here? Maybe better readable and at the same time more obvious later when we clean up the code.